### PR TITLE
blog/content/post/2023-09-12-simplifying-firmware-update.md: addres r…

### PR DESCRIPTION
**Pull request by mkopec**
…eview

Continued from https://github.com/3mdeb/news-and-ideas/pull/538

### CODE REVIEW

File: blog/content/post/2023-09-12-simplifying-firmware-update.md [hunk 1]
```diff
@@ -103,9 +124,14 @@ on-board peripherals.
 
 Capsule Updates are also supported under Linux by the fwupd / LVFS project, as
 well as various BSD derivatives, support for which we helped develop in the
-past few years. TODO link to relevant blogposts
+past few years:
+[fwupd for FreeBSD status update](https://blog.3mdeb.com/2021/2021-06-14-fwupd-freebsd-status/)
```
Comment (BeataZdunczyk) [inline blog/content/post/2023-09-12-simplifying-firmware-update.md:128]: Missing period at the end of the sentence but overall this blog post is well-written and informative! Good job @mkopec.

I did have some corrections to the abstract and title. Sorry, I didn't look into the abstract previously: https://github.com/3mdeb/news-and-ideas/pull/538#discussion_r1338643023, and https://github.com/3mdeb/news-and-ideas/pull/538#discussion_r1338642946.

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

File: blog/content/post/2023-09-12-simplifying-firmware-update.md (hunk 1 above)
Comment (mkopec) [reply blog/content/post/2023-09-12-simplifying-firmware-update.md]: Fixed in d1426a8ca9f2b738428b08d4aef5d07590b382dc

**Comment by mkopec** [comment]
SEO lint checker is failing on unrelated issues :|

**Comment by mkopec** [comment]
Added YT link, rebased to fix CI.

**Comment by mkopec** [comment]
...now it fails because another link is temporarily unavailable 🫠

File: blog/content/post/2023-09-12-simplifying-firmware-update.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
 ---
-title: 'Simplifying Firmware Updates in Dasharo'
-abstract: "Updating your firmware is an important part of keeping your device
-          secure and reliable. Making this process as easy and reliable as
-          possible is, therefore, a big focus for Dasharo. In this article we'll
```
Comment (BeataZdunczyk) [inline blog/content/post/2023-09-12-simplifying-firmware-update.md:5]: Please revert the deletion of the first two sentences. @mkopec

Comment (mkopec) [reply blog/content/post/2023-09-12-simplifying-firmware-update.md]: done 8bfad9d51ab588868e68877d4bb16a3fc3b31a43

**Pull request merged**